### PR TITLE
Added Homepage

### PR DESCRIPTION
**Pull request by HrishikeshMahalle**
HomePage!!
Screenshot (671)

### CODE REVIEW

File: styles.css [hunk 1]
```diff
@@ -0,0 +1,374 @@
...
+    width: 100%;
+    height: 45rem;
+    border: solid;
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    /* background:linear-gradient(transperant, black, 30) url("./Assets/dying-light-2.jfif");
+    background-repeat: no-repeat;
+    background-size: cover;
+    filter: blur(); */
```
Comment (fahd23) [inline styles.css:24]: remove commented code

File: styles.css [hunk 2]
```diff
@@ -0,0 +1,374 @@
...
+    padding: 1rem;
+    font-size: 2rem;
+    
+    gap: 1rem;
+    align-items: center;
+}
+.bg-info-price button{
+    all: unset;
+    font-size: 1.2rem;
+    background-color: #CD3D0F;
```
Comment (fahd23) [inline styles.css:140]: Use CSS variable  to be consistency

File: styles.css [hunk 3]
```diff
@@ -0,0 +1,374 @@
...
+    flex-direction: row;
+    overflow-y:hidden;
+}
+.card-item.horizontal{
+    min-width: 20rem;
+    min-height: 20rem;
+    text-align: center;
+    display: flex;
+    flex-direction: column;
+    background-color: #af4261;
```
Comment (fahd23) [inline styles.css:237]: same here

**Review by fahd23 — APPROVED** [review]
Well structured code, good to merge

File: styles.css (hunk 3 above)
Comment (HrishikeshMahalle) [reply styles.css:237]: yes will change it..

**Review by md-amir-gauhar — APPROVED** [review]
Follow the above suggestions. Else everything looks good

**Pull request merged**